packages/fxa-content-server/app/tests/spec/views/ready.js

```diff
@@ -234,6 +239,28 @@ describe('views/ready', function() {
     });
   });
 
+  describe('isSignedIn', () => {
+    it('shows default `serviceName` if not set by the relier and user is signed in', () => {
+      sinon.stub(relier, 'isSync').callsFake(() => false);
+      assert.equal(
+        view.$(SIGNIN_COMPLETE.SERVICE_NAME).text(),
+        'You are now ready to use Account Settings'
+      );
+    });
+
+    it('shows generic message if `serviceName` is not set by the relier and user is not signed in', () => {
+      user.isSignedInAccount.restore();
+      sinon.stub(user, 'isSignedInAccount').callsFake(() => false);
+
+      return view.render().then(function() {
+        assert.equal(
+          view.$('.account-ready-generic').text(),
+          'Your account is ready!'
+        );
+      });
+    });
+  });
+
```

Comment:
No existing tests for `emailVerified`?  Wanna add some?  😁

Reply:
@chenba Sure thing. I'll create a follow-up PR this morning so this one can get in this train - we also aren't testing that `secondaryEmailVerified` spits out the correct copy and we probably should be.